test/gtest/uct/ib/test_dc.cc

```diff
@@ -448,10 +448,13 @@ UCS_TEST_P(test_dc_flow_control, soft_request)
     EXPECT_EQ(get_fc_ptr(m_e1)->fc_wnd, s_thresh - 1);
 }
 
-/* Check that flush returns UCS_OK even if there is an outgoing grant request */
+/* Check that:
+ * 1) flush returns UCS_OK even if there is an outgoing grant request
+ * 2) No crash when grant for destroyed ep arrives */
```

Comment:
how is this checked?

Reply:
we enable m_e2 entity and send a bunch of messages to m_e1 (which means that grant was sent back as well).
Grant is handled by m_e1, because we check that the last message from m_e2 is handled (checking the counter, which is increased in am callback)